hack/release.sh

```diff
@@ -26,7 +26,9 @@ COMPONENTS=(
   ["gcppubsub.yaml"]="gcppubsub/config"
   ["event-display.yaml"]="config/tools/event-display"
   ["camel.yaml"]="camel/source/config"
-  ["kafka.yaml"]="kafka/source/config"
+  ["kafka-importer.yaml"]="kafka/source/config"
+  ["kafka-channel-crd.yaml"]="kafka/channel/config"
```

Comment:
Can we just call this `kafka-channel.yaml` so it doesn't change names when the CCP goes away?

Reply:
See https://github.com/knative/eventing-contrib/pull/515/files#diff-cede583a27c227d458ca67079957c8a2R30

Reply:
on the master we have it named `kafka-channel`, I felt its better to make it clear here, but happy to rename it